examples/dgmulti_2d/elixir_navierstokes_lid_driven_cavity.jl

```diff
@@ -5,10 +5,10 @@ using Trixi
 # semidiscretization of the ideal compressible Navier-Stokes equations
 
 prandtl_number() = 0.72
-mu = 0.001
+mu() = 0.001
```

Comment:
```suggestion
mu = 0.001
```
Is that needed here? We don't need `mu()` anywhere else (especially not in the initial condition). So it shouldn't make a difference, I think. In that case I think it's easier to keep it a number. The same holds for `prandtl_number` actually.
But this is not so important, I was just wondering.

Reply:
No, I don't think so - I believe this is here due to copy and paste. For manufactured solutions to Navier-Stokes, `initial_condition` depends on `mu`, but because only the inviscid equations are passed in to `initial_condition`, we do not have any way to access`mu` in a type stable way in the initial condition function unless it is a function or another global constant. I'm happy to remove it here.